File: naisc-rest/src/main/java/org/insightcentre/uld/naisc/rest/DatasetNotFoundException.java
package org.insightcentre.uld.naisc.rest;

public class DatasetNotFoundException extends Exception {
    public DatasetNotFoundException() {
    }

    public DatasetNotFoundException(String s) {
        super(s);
    }

    public DatasetNotFoundException(String s, Throwable throwable) {
        super(s, throwable);
    }

    public DatasetNotFoundException(Throwable throwable) {
        super(throwable);
    }
}

File: naisc-rest/src/main/kotlin/org/insightcentre/uld/naisc/rest/DatasetNotFoundException.kt
package org.insightcentre.uld.naisc.rest

class DatasetNotFoundException : Exception {
    constructor() {}

    constructor(s: String) : super(s) {}

    constructor(s: String, throwable: Throwable) : super(s, throwable) {}

    constructor(throwable: Throwable) : super(throwable) {}
}
